src/harness/util.py

```diff
@@ -505,15 +441,20 @@ def areTwoPpasEqual(first_ppa, second_ppa, delta = 0):
  Note: the PPA should contain only one GeoJson Feature, this feature should contain only one Polygon
        and polygons with holes are accepted
   """
-  polygon_of_first_ppa = first_ppa["zone"]["features"][0]["coordinates"]
-  polygon_of_second_ppa = second_ppa["zone"]["features"][0]["coordinates"]
-  if(not _areTwoPolygonsEqual(polygon_of_first_ppa, polygon_of_second_ppa, delta)):
-     return False
+  geometry_of_first_ppa = first_ppa['zone']['features'][0]['geometry']
+  geometry_of_second_ppa = second_ppa['zone']['features'][0]['geometry']
+
+  if not utils.PolygonsAlmostEqual(geometry_of_first_ppa, geometry_of_second_ppa, delta):
+    return False
+  if not utils.hasPolygonCorrectGeoJsonWinding(geometry_of_first_ppa):
```

Comment:
I'm not sure that the check belongs here. You want to throw an error if the SAS gives a polygon which is not CCW, but that is unrelated to the equivalence of two polygons.

Reply:
I don't know If I got your comment correctly, but I added error log for those checks.
when comparing two ppa if one of them is not a "correct" ppa the comparison can return false and the test will fail with logging error, is that ok?

Reply:
What I was trying to say is: This function is supposed to check for the equivalence of two GeoJSON contours. For testing equivalence, it's irrelevant if the winding is CCW or CW.

We need to do the CCW check on the contour, but it doesn't make sense to do it here since this function is concerned only with testing contour equivalence.

Reply:
actually this check is in areTwoPpasEqual and this function is supposed to check if PPAs are equal, so do you prefer that I move this check outside of it?

Reply:
Two PPAs which are identical but with a CW winding would still be considered equal, so it seems logical to me that the output of the function `areTwoPpasEqual()` for those two PPAs should be `True` -- even if both of the contours are invalid. The winding direction is unrelated to whether or not the contours are equal, so it's strange to have it in this function.

You could instead check the winding direction in the loop from line 350 of the ...FAD_testcase.py file (`for ppa_record in ppa_dump_data`).